packages/next-urql/README.md

```diff
@@ -213,6 +213,38 @@ let default = (withUrqlClient(. clientOptions))(. make);
 
 The great part about writing thin bindings like this is that they are zero cost – in fact, the above bindings get totally compiled away by BuckleScript, so you get the full benefits of type safety with absolutely zero runtime cost!
 
+### Restricting Data Fetching to the Server
+
+If you want to use the Urql client in server-side-rendered mode of NextJS you have to load data in `getServerSideProps` lifecycle method. This will load your data while rendering a page on server.
```

Comment:
```suggestion
If you want to use a `Client` in Next.js' newer methods like `getServerSideProps` you may use the `initUrqlClient` method to create a client on the fly. THis will need to be done per request.
```